content/projects/gsoc/2020/project-ideas/jenkins-distribution-customize-service.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: gsocprojectidea
+title: "Jenkins distribution customize service"
+goal: "Provide an out of the box solution for Jenkins users"
+category: Service
+year: 2020
+status: draft
+skills:
+- Java
+- Web Development
+mentors:
+- "linuxsuren"
+---
+
+For many users, they download Jenkins first, then select some plugins and config them. 
+It might takes a lot of time, like hours. But if we can get a perfect Jenkins distribution which contains all we need, 
+it can save that time for us.
+
+The service could like this, a website offered as https://customize.jenkins.io. Plus, it should support self-host. 
+People can select the following configuration items:
+
+* Jenkins core version
+* plugins
+* common configuration, user/password, update-center site e.g.
+* plugin based configuration, Kubernetes, Sonarqube plugin config e.g.
+* multi-form package, jenkins.war or docker image
+* other things
+
+Two projects are expected backend and frontend. Thanks to link:https://github.com/jenkinsci/custom-war-packager[Custom Jenkins WAR packager for Jenkins], 
+it already did a lot of works. It can be the main library for the backend project. 
```

Comment:
```suggestion
it already does a lot of the work . It can be the main library for the backend project. 
```

Reply:
That project has done a lot of the work. So I use `did` instead of `does`. Please correct me if I'm wrong.

Reply:
Not sure how nit picky we want to be in English grammar, but in that case, "it already did a lot of the work" would be correct

Reply:
Yeah not a problem :smile: just a suggestion